airflow/jobs/scheduler_job.py

```diff
@@ -1616,26 +1633,6 @@ def _validate_and_run_task_instances(self, simple_dag_bag: SimpleDagBag) -> bool
         self._process_executor_events(simple_dag_bag)
         return True
 
-    def _process_and_execute_tasks(self, simple_dag_bag):
```

Comment:
Moving code around like this makes PRs harder to review -- as you are mixing two different things here and making more work for reviewers -- and this change is unrelated to "not loading DAGs in main scheduler loop".

Reply:
I will revert this change and propose it in a separate PR.